README.md

```diff
@@ -22,11 +23,7 @@ $ npm install ak-phrasae.js
 ```js
 const sentenceGenerator = require('ak-phrase.js');
 
-const wordArray = [
-    ['eat'],
-    ['code', 'commit'],
-    ['sleep']
-];
+const wordArray = [['eat'], ['code', 'commit'], ['sleep']];
```

Comment:
Any specific reason for this change?
I was kept with line break intentionally, to make it easily readable.

Reply:
Oh no sorry while updating the ReadMe I had made a few accidental changes in the formatting but never reverted the changes my apologies there isn't any specific reason

Reply:
Ok nice